api-reference-v2/mint.json

```diff
@@ -76,7 +76,7 @@
       ]
     },
     {
-      "group": "Merchant Connector Account",
+      "group": "Connector Account",
       "pages": [
         "api-reference/merchant-connector-account/merchant-connector--create",
```

Comment:
Can we update the directory and file names to be "connector account" as well?

Reply:
Will create separate PR for this

Reply:
Done, please check